HW0/P5/P5.txt

```diff
@@ -1 +1,25 @@
-# Your answers here.
+By: Matheus C. Fernandes
+
+Question 1: 255 seconds
+	If one cashier were counting the add the totals of 256 bags it would take 255 seconds because the first count 1+1=2 would count as one secong and every count thereafter would also count as one second 
+
+Question 2: 34 seconds
+	Yes, assuming they imidiatelly divide up the work evenly, they can deivide the 256 bags by 8 workers. Each worker than has to count 32 bags which would take 31 seconds to count. Then 4 workers can add up the other bags, taking 1 additional second. Then 2 workers take the sum of 4 people, taking 1 additional second. Finally, the 1 worker adds the final sum of two people, during the last 1 second.
```

Comment:
I got the same answer is you, but I found the explanation slightly confusing. You might consider putting in a simple asci graphic, i.e. 

x x x x x x x x
 |      |     |    |
 x   x   x   x
     |       |
   x       x
        |
        x

To show the part where they sum up a tree.